### PR TITLE
add in different theme images

### PR DESCRIPTION
**Pull request by andymckay**
an example of https://bugzilla.mozilla.org/show_bug.cgi?id=1329242

### CODE REVIEW

**Comment by andymckay** [comment]
screenshot 2017-10-05 15 48 34
screenshot 2017-10-05 15 48 22

**Comment by andymckay** [comment]
Ah I think that's https://bugzilla.mozilla.org/show_bug.cgi?id=1404568.

File: tabs-tabs-tabs/manifest.json [hunk 1]
```diff
@@ -11,5 +11,13 @@
...
-  "version": "1.0"
+  "version": "1.4",
+  "applications": {
+      "gecko": {
+          "id": "a@b.c"
+      }
+  },
+  "background": {
+      "scripts": ["background.js"]
+  }
```
Comment (wbamberg) [inline tabs-tabs-tabs/manifest.json]: Is the change to this file intentional? It doesn't look related and there's not description of it.

Comment (andymckay) [reply tabs-tabs-tabs/manifest.json]: Totally accident. Will remove.
